samples/legacy_support/All_ID_Mapping_Original.csv

```diff
@@ -2701,17 +2701,17 @@ id,Name,LegacyKey,MainIcon,Modifier1,Modifier2,ExtraIcon,FullFrame,GeometryType,
 2699,Man-Made Structures : Pipelines/Pipe,W---MPA---,46160800,,,,,Line,,,
 2700,Man-Made Structures : Pile/Piling/Post,W-S-MPA---,46160900,,,,,Point,,,
 2701,Signal Intercept (Space) : Communications : Communications : Satellite Downlink (Space),I-P-SCD---,I-P-SCD---,,,,,Point,,,
-2702,Signal Intercept (Space) : Communications,I-A-SCC---,50110100,,,,,Point,,,
-2703,Signal Intercept (Space) : Communications : Communications : Point-to-Point Line of Sight (LOS) (Space),I-A-SCP---,50110100,50491,,,,Point,,,
+2702,Signal Intercept (Space) : Communications,I-A-SCC---,I-S-SCC---,,,,,Point,,,
+2703,Signal Intercept (Space) : Communications : Communications : Point-to-Point Line of Sight (LOS) (Space),I-A-SCP---,I-S-SCP---,,,,,Point,,,
```

Comment:
@csmoore : Same as earlier comment.  I recommend you make a copy of I-S-SCP---.svg and rename it I-A-SCP---.svg.  It should also be Signal Intercept (Air) if the SIDC is I-A-SCC---.